static/third/bootstrap-typeahead/typeahead.js

```diff
@@ -39,6 +39,16 @@
  *   pass the keyup event to the updater.
  *
  *   Our custom changes include all mentions of this.trigger_selection.
+ *
+ * 3. Header text:
+ *
+ *   This adds support for showing a custom header text like: "You are now
+ *   completing a user mention". Provide the function `this.header` that
+ *   returns a string containing the header text, or false.
+ *
+ *   Our custom changes include all mentions of this.header, some CSS changes
+ *   in compose.scss and splitting $container out of $menu so we can insert
+ *   additional HTML before $menu.
```

Comment:
Can you split this commit into first one that adds `$container` and then one adding the feature?  I think that'd be more understandable.